win32/tests/async_socket_win32_int/async_socket_win32_int.c

```diff
@@ -11,10 +11,13 @@
 #include "winsock2.h"
 #include "ws2tcpip.h"
 #include "windows.h"
-#include "azure_macro_utils/macro_utils.h"
+
 #include "testrunnerswitcher.h"
+
+#include "azure_macro_utils/macro_utils.h"
 #include "azure_c_pal/async_socket.h"
 #include "azure_c_pal/execution_engine.h"
+#include "azure_c_pal/gballoc_hl.h"
```

Comment:
>gballoc_hl [](start = 22, length = 10)

redirect header?

Reply:
only needed in context of mocking. Otherwise, gballoc_hl is included because it provides the init_deinit functions which are needed to be called before other memory allocationg routines.

---
In reply to: [462451877](https://github.com/Azure/azure-c-pal/pull/21#discussion_r462451877) [](ancestors = 462451877)